components/todo/Index.vue

```diff
@@ -1,46 +1,43 @@
 <template>
-  <div class="container">
+  <SplashScreen v-if="isSplashScreenVisible" />
+  <div v-else class="container">
     <TodoHeader />
     <TodoContainer />
   </div>
 </template>
 <script>
 import TodoHeader from '@/components/todo/Header.vue'
 import TodoContainer from '@/components/todo/TodoContainer.vue'
+import SplashScreen from '@/components/todo/utils/SplashScreen.vue'
 
 export default {
   name: 'TodoIndex',
   components: {
     TodoHeader,
     TodoContainer,
+    SplashScreen,
   },
   data() {
-    return {}
+    return {
+      isSplashScreenVisible: true,
+    }
+  },
+  mounted() {
+    setTimeout(() => {
+      this.isSplashScreenVisible = false
+    }, 1000)
```

Comment:
can we make this content loading dependent instead of arbitary value?

Reply:
Bhai, this has resolved in the next PR where data is retrieving from real database (supabase) and loader added for initial fetching time.